tests/common/mod.rs

```diff
@@ -302,6 +304,11 @@ pub(crate) fn setup_node(chain_source: &TestChainSource, config: Config) -> Test
 			builder.set_chain_source_bitcoind_rpc(rpc_host, rpc_port, rpc_user, rpc_password);
 		},
 	}
+
+	if let Some(seed) = seed_bytes {
+		builder.set_entropy_seed_bytes(seed).unwrap();
```

Comment:
Unrelated to this PR, but why don't we take `[u8; WALLET_KEYS_SEED_LEN]`? Bindings?